src/cmd/src/error.rs

```diff
@@ -305,6 +305,12 @@ pub enum Error {
         error: std::io::Error,
     },
 
+    #[snafu(display("Failed to spawn thread"))]
+    SpawnThread {
+        #[snafu(source)]
+        error: std::io::Error,
+    },
```

Comment:
Do we need to add `#[cfg(debug_assertions)]` for it?

Reply:
> Do we need to add `#[cfg(debug_assertions)]` for it?

Other function in cmd crate might need it anyway and cfg in enum and match arm is ugly so